lang/en.yml

```
@@ -79,6 +79,7 @@ en:
     MoreOptions: 'More options'
     Published: 'Published {name} {link}'
     Unpublished: 'Unpublished {name} {title}'
+    PUBLISHEDTOASTMESSAGE: 'Published {name} {title}'
```

```suggestion
    PUBLISHEDTOASTMESSAGE: 'Published {type} ''{title}''.'
```
As per the changes in the framework PR. Please also apply to the instance of this in PHP code.

---

Updated